bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/handler/DaikinAcUnitHandler.java

```diff
@@ -312,6 +337,34 @@ private void changeMode(Mode mode) throws DaikinCommunicationException {
         webTargets.setControlInfo(info);
     }
 
+    private void changeHomekitMode(String homekitmode) throws DaikinCommunicationException {
+        ThingTypeUID thingTypeUID = thing.getThingTypeUID();
+        if (homekitmode.equals("off")) {
+           changePower(false);
+        } else {
+           changePower(true);
+           if (homekitmode.equals("on")) {
```

Comment:
Prefer the constant on the left side. So if `homekitmode` is null it will not give an exception.
```suggestion
           if ("on".equals(homekitmode)) {
```

In eclipse put the cursor on equals and click `Ctrl-1`. This will give the option to swap the equal parameters.
This comment applies to all other use cases.